tests/test_cli/test_config.py

```diff
@@ -24,11 +24,12 @@
 import tempfile
 from pathlib import Path
 
+from click.testing import CliRunner
+
 from aea.cli import cli
 from aea.cli.utils.constants import ALLOWED_PATH_ROOTS
-from aea.test_tools.click_testing import CliRunner
 
-from ..conftest import CLI_LOG_OPTION, CUR_PATH
+from tests.conftest import CLI_LOG_OPTION, CUR_PATH
```

Comment:
please could you use relative import? Just for consistency with other test modules (if by any chance you find the non-relative import it should be uniformed to its relative version)